test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpMetricsExporterTests.cs

```diff
@@ -57,7 +57,8 @@ public void ToOtlpResourceMetricsTest(bool includeServiceNameInResource)
 
             using var meter = new Meter($"{Utils.GetCurrentMethodName()}.{includeServiceNameInResource}", "0.0.1");
 
-            var metricReader = new BaseExportingMetricReader(new TestExporter<Metric>(RunTest))
+            var exportedItems = new List<Metric>();
+            var metricReader = new BaseExportingMetricReader(new InMemoryExporter<Metric>(exportedItems))
```

Comment:
Suggestion:
1. pass this directly to the `AddReader`, no need to create `metricReader` variable.
2. change `metricReader.Collect()` to `provider.ForceFlush()`.

Similar code can be found from https://github.com/open-telemetry/opentelemetry-dotnet/blob/5325185d0221e363e30c7daa45610d01f8bfb451/test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusSerializerTests.cs#L43.

Reply:
done, thnx

Reply:
Thanks! 👍